model/riscv_sys_regs.sail

```diff
@@ -207,6 +208,8 @@ function haveZkne()   -> bool = true
 function haveZknd()   -> bool = true
 
 function haveZmmul()  -> bool = true
+function haveRVV()    -> bool = misa.V() == 0b1
```

Comment:
Is this the check we actually want? V is split up into a lot of sub-extensions, I'd expect most to instead be a `haveZvfoo` check (that may well be implemented in terms of misa.V for now as we don't have a good way to control individual Z* extensions yet).

Reply:
I'm not sure about this ... I'll figure it out later.